client/core/trade.go

```diff
@@ -947,19 +948,27 @@ func (t *trackedTrade) processAudit(msgID uint64, audit *msgjson.Audit) error {
 	auth.AuditStamp = audit.Time
 	auth.AuditSig = audit.Sig
 	proof.CounterScript = audit.Contract
+	matchTime := encode.UnixTimeMilli(int64(auth.MatchStamp))
+	reqLockTime := matchTime.Add(dex.LockTimeMaker).Truncate(time.Millisecond) // counterparty = maker, their locktime = 48 hours.
```

Comment:
Shouldn't `reqLockTime` be second resolution, not millisecond?

Also, I feel like both client and server should perhaps be using the same function from `dex` or a common package like `NextSecond` to take a time or a ms stamp and return the next second.  That way they can both be guaranteed to do the same thing with the match stamp.

Reply:
> Shouldn't reqLockTime be second resolution, not millisecond?

Of course it should.

> Also, I feel like both client and server should perhaps be using the same function like `NextSecond` from `dex` or a common package to take a time or a ms stamp and return the next second. That way they can both be guaranteed to do the same thing with the match stamp.

I'm not understanding the use of "next" here. Won't truncate be the "previous" second?

Reply:
"next" meaning the ceil operation or the opposite of truncate, round up I guess.  But as long as everything is consistent it doesn't matter which direction up/down and I see that https://github.com/decred/dcrdex/pull/438/commits/9b39f6f8a46d69d9bd63c03b4507b3b122e75dde makes it super clear now